rasa/nlu/extractors/extractor.py

```diff
@@ -0,0 +1,90 @@
+from typing import Any, Dict, List, Text, Tuple
+
+from rasa.nlu.components import Component
+from rasa.nlu.constants import EXTRACTOR, ENTITIES
+from rasa.nlu.training_data import Message
+
+
+class EntityExtractor(Component):
```

Comment:
Did you change anything in here? I assume not.

Reply:
no, just copy pasted to a new file